### DISCUSSION TITLE
Message 'No module name 'pywintypes' when importing win32com.client

### DISCUSSION THREAD

**Discussion by mwhite62** [IronLanguages/ironpython3#1750] [Q&A]
When I try to run a script that imports win32com.client, I get the error message No module named 'pywintypes'.  If I run the same script directly from the python command line, the script completes successfully.  Is there a problem with win32com.client specifically with IronPython?  I have confirmed I have pywin installed as well as pypiwin installed.

**Comment by slozier** [comment]
I think pywintypes is a C extension module and so doesn't work with IronPython.
